pom.xml

```diff
@@ -28,14 +28,16 @@
         <maven.test.redirectTestOutputToFile>false</maven.test.redirectTestOutputToFile>
 
         <version.io.rest-assured>4.1.2</version.io.rest-assured>
+        <version.javax.servlet.javax.servlet-api>3.0.1</version.javax.servlet.javax.servlet-api>
         <version.junit>4.12</version.junit>
         <version.org.eclipse.microprofile>3.0</version.org.eclipse.microprofile>
+        <version.org.fusesource.jansi>1.18</version.org.fusesource.jansi>
         <version.org.jboss.arquillian>1.5.0.Final</version.org.jboss.arquillian>
         <version.org.jboss.wildfly.dist>18.0.0.Final</version.org.jboss.wildfly.dist>
         <version.org.wildfly.arquillian>2.2.0.Final</version.org.wildfly.arquillian>
-        <version.org.fusesource.jansi>1.18</version.org.fusesource.jansi>
         <version.org.wildfly.core>10.0.3.Final</version.org.wildfly.core>
         <version.org.wildfly.extras.creaper>1.6.1</version.org.wildfly.extras.creaper>
+        <version.org.wildfly.core.wildfly-core-testsuite-shared>11.0.0.Beta3</version.org.wildfly.core.wildfly-core-testsuite-shared>
```

Comment:
Why do we need this ?

Reply:
Basically reload and wait for the server.